docs/en-US/component/table.md

```diff
@@ -258,6 +258,7 @@ table/table-layout
 | default-sort              | set the default sort column and order. property `prop` is used to set default sort column, property `order` is used to set default sort order                                                                                                                               | object                                                                                                                                                               | (order: 'ascending' \| 'descending')                | if `prop` is set, and `order` is not set, then `order` is default to ascending                                          |
 | tooltip-effect            | the `effect` of the overflow tooltip                                                                                                                                                                                                                                        | string                                                                                                                                                               | dark / light                                        | dark                                                                                                                    |
 | tooltip-options ^(2.2.28) | the options for the overflow tooltip, [see the following tooltip component](tooltip.html#attributes)                                                                                                                                                                        | ^[object]`Pick<ElTooltipProps, 'effect' \| 'enterable' \| 'hideAfter' \| 'offset' \| 'placement' \| 'popperClass' \| 'popperOptions' \| 'showAfter' \| 'showArrow'>` | [refer to tooltip](tooltip.html#attributes)         | ^[object]`{ enterable: true, placement: 'top', showArrow: true, hideAfter: 200, popperOptions: { strategy: 'fixed' } }` |
+| append-filter-panel-to    | which element the filter panels appends to.                                                                                                                                                                                                                                 | string                                                                                                                                                               | —                                                   | body                                                                                                                    |
```

Comment:
@btea I assume I should add version tags here too, is it `^(2.8.3)`?

Reply:
Yes.